app/views/layouts/admin/base.html.erb

```diff
@@ -0,0 +1,11 @@
+<% content_for :nav_bar do %>
+  <%= render PrimaryNavComponent.new do |c| %>
```

Comment:
Let's use full words. What is `c`? `component`?

Reply:
Copied directly from existing components

Reply:
They are wrong too! 😂 

I must have missed it there :P